website/content/sdk/installation.md

```diff
@@ -83,8 +94,16 @@ When using a minimal installation, enter:
 pip install -U openbb
 ```
 
-When using a full installation, with the `obb` Python environment active, enter:
+When using a an installation with toolkits, with your virtual environment activated, enter:
 
 ```shell
 pip install -U openbb[all]
 ```
+
+## Nightly Builds
+
+OpenBB SDK is updated daily with new features and bug fixes, but some features being worked on may be unstable. To use the same SDK version as the development team, install the nightly build with:
+
```

Comment:
Again should include steps about activating the environment. Or is it fine to install this one on the base environment? I know it isn't but it doesn't say that.

Reply:
addressed this. moved to pypi section of the terminal docs and specified how to activate the env